README.md

```diff
@@ -634,8 +634,46 @@ docker run --volumes-from dcert -e DOCKER_HOST=$DOCKER_HOST -e DOCKER_TLS_VERIFY
 Different CI/CD services have different containerized environment designs that impose various restrictions that may impact the ability of the main app to communicate with the sensor app embedded in the temporary container Slim creates. Try adjusting the values for the `--sensor-ipc-mode` and `--sensor-ipc-endpoint` flags. This [`Google Cloud Build`](https://medium.com/google-cloud/integrating-dockerslim-container-minify-step-on-cloud-build-64da29fd58d1) blog post by Márton Kodok is a good reference for both of those flags. 
 
 ### Using `*-file` Flags
+- There are several flags that accept file paths (`--include-path-file`, `--compose-file`, `--http-probe-cmd-file`, etc). You need volume mount the location of the referenced paths or the file paths themselves when you use the containerized version of Slim because the Slim app container won't have accept to the referenced files otherwise.
 
-There are several flags that accept file paths (`--include-path-file`, `--compose-file`, `--http-probe-cmd-file`, etc). You need volume mount the location of the referenced paths or the file paths themselves when you use the containerized version of Slim because the Slim app container won't have accept to the referenced files otherwise.
+### CI/CD INTEGRATIONS
+
+#### Integrating Slim in Jenkins
+Given Jenkins is running as a containerized environment in a virtual machine(AWS EC2 Instance), the following example steps highlight integrating Slim in Jenkins: 
+- Make Docker commands available in Jenkins
+```
+docker run -p 8080:8080 -p 50000:50000 -d \  
+-v jenkins_home:/var/jenkins_home \ 
+-v /var/run/docker.sock:/var/run/docker.sock \ 
+-v $(which docker):/usr/bin/docker jenkins/jenkins:lts
+```
+- Enable Docker permissions
+```
+chmod 666 /var/run/docker.sock 
+```
+- Pull Slim Image 
+```
+docker pull dslim/slim
+```
+- Building an Image and Executing Slim commands
+```
+docker build -t node_alpine:$BUILD_NUMBER .
+```
+```
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock dslim/slim \
+build --target node_alpine:$BUILD_NUMBER --tag node_alpine:slim-$BUILD_NUMBER \
+exit
+```
+
+#### Integrating Slim in Github Actions
+Github Actions has Runners running docker in the background, the following example steps highlight integrating slim in Github Actions using [Docker-Slim Github Action](https://github.com/marketplace/actions/docker-slim-github-action):
+- Building an Image and Executing Slim commands 
+```
+docker build -t node_alpine:${{github.run_number}} .
+```  
+```
+slim build --target node_alpine:${{github.run_number}} -tag slim-${{github.run_number}}
```

Comment:
Not clear what both of these snippets represent... They look like regular CLI commands you'd run in your terminal. What do they have to do with Github Actions?

I expected to see a small paragraph or, at least, a sentence that describes how to use the Github Action you referenced. And I expected to see a Github Action code snippet like this (instructing users to insert something like this in their Github Action):

```
      # Slim it!
      - uses: kitabisa/docker-slim-action@v1
        env:
          DSLIM_HTTP_PROBE: false
        with:
          target: ${{ github.repository }}:latest
          tag: "slim"
```

Reply:
Hello @kcq, thanks for the feedback, I have rewritten the entire CI/CD integrations section in the repo README, kindly review.

Reply:
Thanks a lot! Good to have the config file snippets for the CI/CD integrations.